### PR TITLE
Install ios docs add pod line and updated.

### PR DESCRIPTION
**Pull request by yasaricli**

### CODE REVIEW

File: docs/ios-installation.md [hunk 1]
```diff
@@ -14,9 +14,17 @@ In addition to the standard React Native requirements, you will also need the fo
 
 ## Install react-native-azurenotificationhub
 
-```
-npm install react-native-azurenotificationhub
-```
+Install the library using either **Yarn**:
+
+    yar add react-native-azurenotificationhub
```
Comment (phongcao) [inline docs/ios-installation.md]: Do you mean `yarn add`?

Comment (yasaricli) [reply docs/ios-installation.md]: https://classic.yarnpkg.com/en/docs/cli/add/

for example:

https://github.com/react-native-community/react-native-netinfo#getting-started

Comment (yasaricli) [reply docs/ios-installation.md]: Sorry, I fixed it @phongcao

**Comment by phongcao** [comment]
Thanks for your contribution, @yasaricli!

**Pull request merged**